packages/wallets/src/components/Base/ModalStepWrapper/ModalStepWrapper.scss

```diff
@@ -66,6 +66,12 @@
     }
 
     &__body {
+        height: 100%;
+
+        @include desktop {
+            max-height: calc(var(--wallets-vh, 1vh) * 100 - 10rem);
+        }
```

Comment:
I have already added this css rule at the root just wondering why are we repeating it again here 🤔

Reply:
Hi @aizad-deriv ,
Yes I have seen the one in the root, but I had to do this to prevent the body from overflowing.
Idk why the root one is not working as expected 🤔.